File: src/main/java/com/alquiler/proyecto/entity/Multa.java
package com.alquiler.proyecto.entity;

import java.time.LocalDate;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.JoinColumn;
import jakarta.persistence.ManyToOne;
import lombok.Data;

@Entity
@Data
public class Multa {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long idMulta;

    @ManyToOne
    @JoinColumn(name = "alquiler_id")
    private Alquiler alquiler;

    private String motivo;
    private Double monto;
    private LocalDate fecha;
}

File: backendPython/models/multa.py
from sqlalchemy import Column, Integer, String, Float, Date, ForeignKey
from . import db

class Multa(db.Model):
    __tablename__ = 'multa'
    id_multa = Column(Integer, primary_key=True)
    alquiler_id = Column(Integer, ForeignKey('alquiler.id_alquiler'))
    motivo = Column(String)
    monto = Column(Float)
    fecha = Column(Date)
